windows/src/engine/keyman32/K32_load.cpp

```diff
@@ -75,17 +69,17 @@ BOOL GetKeyboardFileName(LPSTR kbname, LPSTR buf, int nbuf)
   return n;
 }
 
-BOOL LoadlpKeyboardCore(int i)
+BOOL LoadlpKeyboard(int i)
 {
-  SendDebugMessageFormat(0, sdmLoad, 0, "LoadlpKeyboardCore: Enter ---");
+  SendDebugMessageFormat(0, sdmLoad, 0, "LoadlpKeyboard: Enter ---");
```

Comment:
You could use `__FUNCTION__` instead of hardcoding the function name:

```suggestion
  SendDebugMessageFormat(0, sdmLoad, 0, "%s: Enter ---", __FUNCTION__);
```